WindowsServerDocs/storage/storage-spaces/delimit-volume-allocation.md

```diff
@@ -5,16 +5,16 @@ ms.manager: eldenc
 ms.technology: storage-spaces
 ms.topic: article
 author: cosmosdarwin
-ms.date: 02/28/2018
+ms.date: 03/03/2018
 ---
 
 # Delimit the allocation of volumes in Storage Spaces Direct
-> Applies To: Windows Server Insider Preview
+> Applies To: Windows Server Insider Preview, build 17093 and later
 
 Windows Server Insider Preview introduces an option to manually delimit the allocation of volumes in Storage Spaces Direct. Doing so can significantly increase fault tolerance under certain conditions, but imposes some added management considerations and complexity. This topic explains how it works and gives examples in PowerShell.
 
    > [!IMPORTANT]
-   > This feature is new in Windows Server Insider Preview only. It is not available in Windows Server 2016.
+   > This feature is new in Windows Server Insider Preview, build 17093 and later. It is not available in Windows Server 2016.
```

Comment:
Can you insert this text here somewhere? 
We invite IT pros to join the [Windows Insider Program](http://insider.windows.com/) to give us feedback on what we can do to make Windows work better for your organization

Reply:
Done!